app/classifier/tasks/survey/chooser.cjsx

```diff
@@ -11,6 +16,27 @@ module.exports = React.createClass
     onFilter: Function.prototype
     onChoose: Function.prototype
 
+  componentWillMount: ->
+    # refs for the choices
+    @choiceButtons = []
```

Comment:
I went back and re-read [the docs for refs](https://facebook.github.io/react/docs/refs-and-the-dom.html), and now I think `@choiceButtons` is only calculated on mount.